main.js

```diff
@@ -22,6 +22,9 @@ const logSymbols = require('log-symbols')
 const { paramCase } = require('param-case')
 const { pascalCase } = require('pascal-case')
 
+const init = require('@react-native-community/cli/build/commands/init/init')
+  .default
```

Comment:
✅ Now extracted. This works if I would do `yarn add github:brodybits/react-native-init-func`:

```suggestion
const init = require('react-native-init-func')
```